src/components/ResistanceLogo/ResistanceLogo.js

```diff
@@ -1,27 +1,15 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
 
-const ResistanceLogo = (props) => {
+import styles from './ResistanceLogo.sass';
+
+const ResistanceLogo = () => {
   const image = '../static/img/Logo-large.png';
-  const style = {
-    maxWidth: `${props.width}`,
-    height: `${props.height}`
-  };
 
   return (
-    <div style={style}>
-      <img src={image} style={{ width: '100%' }} alt="Logo for the Resistance Calendar" />
+    <div>
+      <img src={image} className={styles.resistanceLogo} alt="Logo for the Resistance Calendar" />
```

Comment:
I have the same question about this as with the AddEventButton component changes -- don't we want to keep the components flexible by having them dictate their own max-width?  If we want this or the AddEventButton to be used in different contexts, this makes it harder.